src/main/api/ledger/address.ts

```diff
@@ -10,7 +10,8 @@ import { getAddress as getTHORAddress } from './thorchain/address'
 
 export const getAddress = async ({
   chain,
-  network
+  network,
+  walletIndex = '0'
```

Comment:
Use `walletIndex: number` everywhere as we do in `xchain-*` (e.g. here https://github.com/xchainjs/xchainjs-lib/blob/c17c625b666c0113779db62db2f585c076be587d/packages/xchain-client/src/types.ts#L55)